src/view/pages/__snapshots__/gettingStarted.spec.tsx.snap

```diff
@@ -59,7 +59,7 @@ exports[`GettingStartedPage component  should render correctly 1`] = `
     <h3
       className="normalFontWeight"
     >
-      &gt; Import the micro:bit library to use it (This is required)
+      Import the Circuit Playground library to use it (This is required)
```

Comment:
It seems that we are keeping the '>' symbol at the start of the line in other lines. Perhaps we keep it here as well?

Reply:
Yep, will re-add it. im making a new PR with correct branch name